File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int left = 0, right =0;
        for(int i=0; i<nums.length; i++){
            if(nums[left]!=nums[right]){
                left++;
                nums[left]=nums[right];
                right++;
            } else {
                right++;
            }
        }
        return left+1;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l = 0
        for r in range(0, len(nums)):
            if nums[l] == nums[r]:
                continue
            else:
                nums[l+1] = nums[r]
                l = l+1;
        return l+1